tests/tekton-pipelines_test.go

```diff
@@ -6,157 +6,258 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
-	"github.com/kubevirt/tekton-tasks-operator/pkg/common"
-	tektontasks "github.com/kubevirt/tekton-tasks-operator/pkg/tekton-tasks"
 	pipeline "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	v1 "k8s.io/api/core/v1"
 	rbac "k8s.io/api/rbac/v1"
+	ssp "kubevirt.io/ssp-operator/api/v1beta1"
+	"kubevirt.io/ssp-operator/internal/common"
+	tekton_tasks "kubevirt.io/ssp-operator/internal/operands/tekton-tasks"
```

Comment:
`tekton_tasks` package does not exist yet in this commit. Please remove the import, so it can compile.

Reply:
That's really weird that it exists here. I don't know how I missed it.